extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -190,6 +191,47 @@ function () {
 		return $botIds;
 	}
 
+
+	/**
+	 * @return array list of blocked ids for Top Contributors
+	 */
+	private function getBlockedIds(){
```

Comment:
I'd consider renaming this method. "Blocked" has a specific meaning in MediaWiki when relating to users, so when I read `getBlockedIds`, I expected the method to be getting a list of "blocked" users (i.e. blocked from editing), not a blacklist of users based on user group.